### PR TITLE
Add missing validation for image compression control

### LINKED ISSUE #6728: Missing VK_EXT_image_compression_control VUs
VUs missing from `VK_EXT_image_compression_control`

- [ ] VUID-VkImageCreateInfo-pNext-06743
- [ ] VUID-VkImageCreateInfo-pNext-06744
- [ ] VUID-VkSwapchainCreateInfoKHR-pNext-06752

### PR DESCRIPTION
**Pull request by ziga-lunarg**
Closes #6728

### CODE REVIEW

File: layers/core_checks/cc_image.cpp [hunk 1]
```diff
@@ -625,6 +625,28 @@ bool CoreChecks::PreCallValidateCreateImage(VkDevice device, const VkImageCreate
...
+                                 ") is not equal to the number of planes in the multi-planar format %s (%" PRIu32 ")",
+                                 image_compression_control->compressionControlPlaneCount, string_VkFormat(pCreateInfo->format),
+                                 vkuFormatPlaneCount(pCreateInfo->format));
+            }
+        } else {
+            if (image_compression_control->compressionControlPlaneCount != 1) {
+                skip |= LogError("VUID-VkImageCreateInfo-pNext-06744", device, create_info_loc,
+                                 "VkImageCompressionControlEXT::flags contain VK_IMAGE_COMPRESSION_FIXED_RATE_EXPLICIT_EXT and "
+                                 "VkImageCreateInfo::format is a not a multi-planar format (%s), but "
+                                 "VkImageCompressionControlEXT::compressionControlPlaneCount is %" PRIu32,
```
Comment (spencer-lunarg) [inline layers/core_checks/cc_image.cpp:644]: we don't need 

```suggestion
                                 "VkImageCompressionControlEXT::compressionControlPlaneCount is %" PRIu32 "",
```

?  (if not, looks good to me)

**Review by spencer-lunarg — APPROVED** [review]

File: layers/core_checks/cc_image.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply layers/core_checks/cc_image.cpp:644]: What would the empty string do?

Comment (spencer-lunarg) [reply layers/core_checks/cc_image.cpp:644]: I guess I didn't know you could end the string with a `PRIu32`

Comment (ziga-lunarg) [reply layers/core_checks/cc_image.cpp:644]: `PRIu32` is just a macro for a string

Comment (spencer-lunarg) [reply layers/core_checks/cc_image.cpp:644]: #TheMoreYouKnow - thanks

**Pull request merged**